src/iolStateMachineHandler/src/module.cpp

```diff
@@ -2084,6 +2090,12 @@ void Manager::updateMemory()
                     list_3d.addString("position_3d");
                     list_3d.addList().read(x);
 
+                    // prepare class_score property
+                    Bottle class_score;
+                    Bottle &list_score=class_score.addList();
+                    list_score.addString("class_score");
+                    list_score.addDouble(scoresMap.at(object));
```

Comment:
you could push `score` straight away here.

Reply:
`score` does not exist anymore in this scope

Reply:
Yep sorry, I didn't realize there were two different scopes from the hunks displayed within the GitHub PR dashboard!